app/controllers/public.js

```diff
@@ -1,24 +1,28 @@
 import Controller from '@ember/controller';
 import moment from 'moment';
-import { computed } from '@ember/object';
+import { action, computed } from '@ember/object';
 
-export default Controller.extend({
-  smallLead: computed('session.currentRouteName', function() {
-    if (this.get('session.currentRouteName')) {
-      return this.get('session.currentRouteName') !== 'public.index';
+export default class extends Controller {
+  @computed('session.currentRouteName')
+  get smallLead() {
+    if (this.session.currentRouteName) {
+      return this.session.currentRouteName !== 'public.index';
     }
-  }),
-  displayEndDate: computed('model.startsAtDate', 'model.endsAtDate', function() {
+    return null;
+  }
+  @computed('model.startsAtDate', 'model.endsAtDate')
+  get displayEndDate() {
     return !moment(this.model.startsAtDate).isSame(this.model.endsAtDate, 'minute');
-  }),
-  displaySideMenu: computed('session.currentRouteName', function() {
+  }
+  @computed('session.currentRouteName')
+  get displaySideMenu() {
     if (this.session.currentRouteName) {
       return this.session.currentRouteName !== 'public.cfs.new-session' && this.session.currentRouteName !== 'public.cfs.new-speaker' && this.session.currentRouteName !== 'public.cfs.edit-speaker' && this.session.currentRouteName !== 'public.cfs.edit-session';
     }
-  }),
-  actions: {
-    toggleMenu() {
-      this.toggleProperty('isMenuOpen');
-    }
+    return null;
```

Comment:
Why this line

Reply:
it was showing error that function``` displaySideMenu ``` should always return .

Reply:
Can we remove the above `if` conditional?

Reply:
yes , i think we can remove

Reply:
> Can we remove the above `if` conditional?

checked after removing if -
![Screenshot from 2019-12-25 10-15-46](https://user-images.githubusercontent.com/56407566/71432440-ba293480-26ff-11ea-8173-3e4b668bdd81.png)
![Screenshot from 2019-12-25 10-16-16](https://user-images.githubusercontent.com/56407566/71432441-bac1cb00-26ff-11ea-9a16-26d90c72138c.png)

Reply:
If it is working then commit the changes :+1: